logger.c

```diff
@@ -137,6 +137,19 @@ static void _logger_log_item_store(logentry *e, const entry_details *d, const vo
     e->size = sizeof(struct logentry_item_store) + nkey;
 }
 
+static void _logger_log_item_deleted(logentry *e, const entry_details *d, const void *entry, va_list ap) {
+    item *it = (item *)entry;
+    int comm = va_arg(ap, int);
+    struct logentry_deletion *le = (struct logentry_deletion *) e->data;
+    le->nkey = it->nkey;
+    le->cmd = comm;
+    le->it_flags = it->it_flags;
```

Comment:
don't think you need to store it_flags since it's not being used in the parse side?

Reply:
you're right - will remove